e2e/suites/search/search-results-files-folders.test.ts

```diff
@@ -113,7 +113,7 @@ describe('Search results - files and folders', () => {
     expect(await dataTable.getSearchResultNameAndTitle(file)).toBe(`${file} ( ${fileTitle} )`);
     expect(await dataTable.getSearchResultDescription(file)).toBe(fileDescription);
     expect(await dataTable.getSearchResultModified(file)).toBe(`Modified: ${modifiedDate} by ${modifiedBy} | Size: ${size} Bytes`);
-    expect(await dataTable.getSearchResultLocation(file)).toBe('Location: Personal Files');
+    expect(await dataTable.getSearchResultLocation(file)).toBe('Location:\nPersonal Files');
```

Comment:
Looks like trying to adopt the test for current layout?

Reply:
we have this situation in other tests as well... where the DOM has tags containing text, the getText() from the test treats that as an empty line...

Reply:
I think we need to improve the CSS selector and check for the exact element, not to rely on how protractor combines the text.